instructor/ui/instructor.js

```diff
@@ -90,7 +90,26 @@ $(function(){
     */
 
     $('.scb_ab_f_select_variable>input').click(function(){
-        select_variables();
+        /* Want to warn the instructor*/
+        var $checkbox= $(this);
+        /* Var created is initialized in select_variables template */
+        if (created === "True") {
```

Comment:
After you make the changes to `created` above you can make this `=== true` instead, or just `if (created)` if you prefer